packages/openapi3/src/lib.ts

```diff
@@ -399,6 +399,13 @@ export const $lib = createTypeSpecLibrary({
         default: paramMessage`Invalid key '${"value"}' used in a fixed field of the Component object. Only alphanumerics, dot (.), hyphen (-), and underscore (_) characters are allowed in keys.`,
       },
     },
+    "sse-not-supported": {
```

Comment:
regarding other comment as well this might be better as a more generic streams not supported